From: Daniel van Vugt <daniel.van.vugt@canonical.com>
Date: Tue, 7 Nov 2023 16:30:01 +0200
Subject: kms/crtc: Increase default deadline evasion to 1000 microseconds

This seems to be enough to fix cursor stutter on X1 Carbon 10 (OLED)
where the display is 90Hz and the touchpad ~145Hz.

Bug: https://gitlab.gnome.org/GNOME/mutter/-/issues/3146
Bug-Ubuntu: https://launchpad.net/bugs/2040977
Origin: https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/3373
Forwarded: yes
---
 src/backends/native/meta-kms-crtc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/backends/native/meta-kms-crtc.c b/src/backends/native/meta-kms-crtc.c
index b17e846..d6a8228 100644
--- a/src/backends/native/meta-kms-crtc.c
+++ b/src/backends/native/meta-kms-crtc.c
@@ -28,7 +28,7 @@
 #include "backends/native/meta-kms-update-private.h"
 #include "backends/native/meta-kms-utils.h"
 
-#define DEADLINE_EVASION_US 800
+#define DEADLINE_EVASION_US 1000
 #define DEADLINE_EVASION_WITH_KMS_TOPIC_US 1000
 
 typedef struct _MetaKmsCrtcPropTable
